### ISSUE TITLE
Remove deprecated t.m.smtp.sendEmail

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4529]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)| @cyli reported|
|-|-|
|Trac ID|trac#4529|
|Type|task|
|Created|2010-06-29 21:25:36Z|
|Branch|https://github.com/twisted/twisted/tree/remove-smtp-sendEmail-4529|

Currently, twisted.news.database still uses sendEmail, but once that ticket ([#4528](https://github.com/twisted/twisted/issues/4528)) is closed twisted.mail.smtp.sendEmail can probably be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__4529 4529
type__task task
reporter__cyli cyli
priority__normal normal
milestone__ 
branch__branches_remove_smtp_sendEmail_4529 branches/remove-smtp-sendEmail-4529
branch_author__cyli cyli
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1277846736000000 1277846736000000
changetime__1288145592000000 1288145592000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_1"></a>|@cyli commented|
|-|-|

(In [30167]) Branching to 'remove-smtp-sendEmail-4529'

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli commented|
|-|-|

(In [30168]) Removed sendEmail from smtp.py.  Also removed some unused imports (cred is
imported, but cred.credentials and cred.checkers were then also imported).

Refs [#4529](https://github.com/twisted/twisted/issues/4529)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli commented|
|-|-|

(In [30169]) Added a topfile/news file.

Refs [#4529](https://github.com/twisted/twisted/issues/4529)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)|@cyli removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli commented|
|-|-|

[build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-smtp-sendEmail-4529)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_6"></a>|@cyli commented|
|-|-|

(In [30180]) Changed newsfile to conform with accepted format.

Refs [#4529](https://github.com/twisted/twisted/issues/4529)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set owner to @cyli|
|-|-|

Thanks for all this cruft-removal, cyli. While you're removing unused imports, pyflakes also shows

twisted/mail/smtp.py:1400: local variable 'e' is assigned to but never used

Other than that, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_9"></a>|@cyli commented|
|-|-|

(In [30181]) Remove an extra variable declaration

Refs [#4529](https://github.com/twisted/twisted/issues/4529)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_10"></a>|@cyli commented|
|-|-|

Removals are easy to do.  :)  Thanks for the review!  Merging.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_11"></a>|@cyli set status to `closed`|
|-|-|

(In [30182]) Merge remove-smtp-sendEmail-4529: Remove deprecated twisted.mail.smtp.sendEmail

Author: cyli
Reviewer: jesstess
Fixes: [#4529](https://github.com/twisted/twisted/issues/4529)

Remove deprecated twisted.mail.smtp.sendEmail

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
